cmd/config.go

```diff
@@ -49,9 +50,21 @@ func newConfigCommand() *cobra.Command {
 		Args:  cobra.ExactArgs(1),
 	}
 
+	migrateCommand := &cobra.Command{
+		Use:                "migrate",
+		Short:              "migrate a pre-release 2.0 config to the official release version",
+		RunE:               migrateConfig,
+		Hidden:             true,
+		DisableFlagParsing: true,
+	}
+	// These flags are for documentation and not actually parsed
+	migrateCommand.PersistentFlags().StringP("config", "c", ".circleci/config.yml", "path to config file (default \".circleci/config.yml\")")
```

Comment:
I believe the `(default ...)` is automatically added by cobra, so it would be included twice if it's in the help text.

Reply:
Nice catch, here's what it would have been :face_with_head_bandage: 

```
  -c, --config string   path to config file (default ".circleci/config.yml") (default ".circleci/config.yml")
```